composer/core/precision.py

```diff
@@ -78,7 +78,8 @@ def get_precision_context(precision: Union[str, Precision],
                 }
             fp8_recipe = DelayedScaling(**precision_config)
             with te.fp8_autocast(enabled=True, fp8_recipe=fp8_recipe):
-                yield
+                with te.onnx_export(enabled=True):
```

Comment:
@j316chuck : can you add a comment here why onnx_export is necessary? People reading this code in future will get confused.

Reply:
Yes that is a good point. Let me do that